### PR TITLE
Enable display of custom commands in @commands.

### PR DESCRIPTION
**Pull request by Emistry**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Based on @Dastgir's commit. https://github.com/RagEmu/Renewal/commit/9409e435cab3c7ee24172f09b865b2803411f0ff
only GM with the minimum group id able to view it.
There was an Issue created in Herc, but I didnt found where was it.

**Affected Branches:** Master

**Preview**
Image

---
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1612)

### CODE REVIEW

File: src/map/atcommand.c [hunk 1]
```diff
@@ -8366,6 +8366,36 @@ void atcommand_commands_sub(struct map_session_data* sd, const int fd, AtCommand
 	dbi_destroy(iter);
 	clif->message(fd,line_buff);
 
+	if (atcommand->binding_count) {
```
Comment (MishimaHaruna) [inline src/map/atcommand.c]: according to the code style guidelines, this should be `atcommand->binding_count > 0`

File: src/map/atcommand.c [hunk 2]
```diff
@@ -8366,6 +8366,36 @@ void atcommand_commands_sub(struct map_session_data* sd, const int fd, AtCommand
...
+		int i, count_bind = 0;
+		int gm_lvl = pc_get_group_level(sd);
+		size_t slen;
+		for (i = 0; i < atcommand->binding_count; i++) {
+			if (gm_lvl >= ((type == COMMAND_ATCOMMAND) ? atcommand->binding[i]->group_lv : atcommand->binding[i]->group_lv_char)) {
+				slen = strlen(atcommand->binding[i]->command);
```
Comment (MishimaHaruna) [inline src/map/atcommand.c]: can you move the `size_t slen` variable declaration here? Right now it's outside the `for` for no reason

File: src/map/atcommand.c [hunk 3]
```diff
@@ -8366,6 +8366,36 @@ void atcommand_commands_sub(struct map_session_data* sd, const int fd, AtCommand
...
+					cur = line_buff;
+					memset(line_buff, ' ', CHATBOX_SIZE);
+					line_buff[CHATBOX_SIZE - 1] = 0;
+				}
+				memcpy(cur, atcommand->binding[i]->command, slen);
+				cur += slen + (10 - slen % 10);
+				count_bind++;
+			}
+		}
+		if (count_bind)
```
Comment (MishimaHaruna) [inline src/map/atcommand.c]: `count_bind > 0`

**Comment by Emistry** [comment]
@MishimaHaruna  changes applied.

File: src/map/atcommand.c (hunk 1 above)
Comment (Emistry) [inline src/map/atcommand.c]: changes applied.

File: src/map/atcommand.c (hunk 2 above)
Comment (Emistry) [inline src/map/atcommand.c]: changes applied.

File: src/map/atcommand.c (hunk 3 above)
Comment (Emistry) [inline src/map/atcommand.c]: changes applied.

**Comment by MishimaHaruna** [comment]
Merged, thank you!

**Pull request merged**